parametric/test_headers_tracestate_dd.py

```diff
@@ -433,6 +434,8 @@ def test_headers_tracestate_dd_propagate_propagatedtags(test_agent, test_library
     dd_items4 = tracestate4["dd"].split(";")
     assert "traceparent" in headers4
 
+    # TODO(stephen.belanger): not sure what the correct behaviour here is,
+    # but Node.js seems to produce an empty string rather than None.
     if headers4["x-datadog-tags"] is None:
```

Comment:
I think an empty `x-datadog-tags` is fine and should be treated as "no x-datadog-tags". How about we change to this?

```suggestion
    if headers4["x-datadog-tags"] is None or headers4["x-datadog-tags"] == "":
```